files/manifests/k8s-controller-manager.yaml

```diff
@@ -32,7 +32,7 @@ spec:
     - --terminated-pod-gc-threshold=10
     - --use-service-account-credentials=true
     - --kubeconfig=/etc/kubernetes/kubeconfig/controller-manager.yaml
-    - --feature-gates=TTLAfterFinished=true,CSIMigrationAWS={{ .EnableCSIMigrationAWS }},CSIMigrationAWSComplete={{ .EnableCSIMigrationAWSComplete }}
+    - --feature-gates=TTLAfterFinished=true,CSIMigrationAWS={{ .EnableCSIMigrationAWS }},InTreePluginAWSUnregister={{ .InTreePluginAWSUnregister }}
```

Comment:
Do we actually still need this flag? I am trying to understand what is stopping us from eventually dropping these :D

Reply:
Customer might still use an old storage class which uses the in-tree plugin to create EBS volumes.

CSIMigrationAWS will forward EBS traffic to EBS CSI Driver in case customer requests it from old in-tree plugin.

When InTreePluginAWSUnregister enabled it will reject sending traffic to in-tree plugin. In case customer doesn't change storage-class it might be a problem.

I already applied with ebs-csi-driver a new storage class `gp3` which will be as the default storage class in release `v15.0.0`. So in case customer doesn't specify the storage-class we are safe, but there might be a custom storage classes from customer still using in-tree plugin.

We should give customers some time and at some point we can drop in-tree.

Reply:
I see - that makes sense to me.

Would be cool if there was even a metric to show how many customers are using the InTreePlugin thing. But maybe something for the future :)